<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Selection of a sphere-bounded sub-network</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonInFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Input file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInFile">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutFile">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOutFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Output file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonZoom">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>310</y>
      <width>491</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>451</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCmgui">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Create CMGUI files</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>963</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonInFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>inFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonOutFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>outFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>18</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonZoom</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>zoomer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonClickHandler()</slot>
  <slot>inFileSelecter()</slot>
  <slot>outFileSelecter()</slot>
  <slot>zoomer()</slot>
  <slot>radioButtonHandler()</slot>
 </slots>
</ui>
